js/snippetPreview.js

```diff
@@ -847,8 +847,8 @@ SnippetPreview.prototype.validateFields = function() {
 SnippetPreview.prototype.updateProgressBars = function() {
 	var metaDescriptionRating, titleRating, metaDescription, title;
 
-	metaDescription = getAnalyzerMetaDesc.call( this );
-	title = getAnalyzerTitle.call( this );
+	metaDescription = YoastSEO.getStringHelper().stripSpaces( getAnalyzerMetaDesc.call( this ) );
```

Comment:
Please move the `stripSpaces` function to the `getAnalyzerMetaDesc` function, this makes sure it's always applied.